src/cache/s3.rs

```diff
@@ -46,19 +43,8 @@ pub struct S3Cache {
 impl S3Cache {
     /// Create a new `S3Cache` storing data in `bucket`.
     pub fn new(bucket: &str, endpoint: &str) -> Result<S3Cache> {
-        let user_dirs = UserDirs::new().ok_or("Couldn't get user directories")?;
-        let home = user_dirs.home_dir();
-
-        let profile_providers = vec![
-            ProfileProvider::with_configuration(home.join(".aws").join("credentials"), "default"),
-            //TODO: this is hacky, this is where our mac builders store their
-            // credentials. We should either match what boto does more directly
-            // or make those builders put their credentials in ~/.aws/credentials
-            ProfileProvider::with_configuration(home.join(".boto"), "Credentials"),
-        ];
```

Comment:
This change is the one bit that I can see being a problem.  The Rusoto client no longer supports configuring multiple `ProfileProvider`s for a single `ChainProvider`, I suspect it's not too hard to implement a local version, but if you're planning to move your credentials anyway, maybe this is your chance?

Reply:
I'm no longer at Mozilla but I suspect this is not a problem anymore because we had switched away from using native mac build machines while I was still there.